modules/apps/asset/asset-list-api/src/main/java/com/liferay/asset/list/util/comparator/AssetListEntryUsageModifiedDateComparator.java

```diff
@@ -23,7 +23,9 @@ public class AssetListEntryUsageModifiedDateComparator
 
 	public static final String[] ORDER_BY_FIELDS = {"modifiedDate"};
 
-	public static AssetListEntryUsageModifiedDateComparator getInstance(boolean ascending) {
+	public static AssetListEntryUsageModifiedDateComparator getInstance(
```

Comment:
@victorhcunha This change should be in the previous commit, and be logged in the replacement?

Reply:
Thanks, @dantewang !
This sf apply really should be in the previous commit.